src/main/scala/stdlib/Lists.scala

```diff
@@ -90,50 +90,93 @@ object Lists extends FlatSpec with Matchers with org.scalaexercises.definitions.
     b should equal(List(res0, res1, res2, res3))
   }
 
-  /** Lists have many useful utility methods:
-   */
-  def usefulMethodsLists(res0: Int, res1: List[Int], res2: List[Int], res3: List[Int]) {
+  /** Lists have many useful utility methods.
+    *
+    * `reverse` return a reversed copy of the list:
+    */
+  def reverseLists(res0: Int, res1: List[Int]) {
     val a = List(1, 3, 5, 7, 9)
 
     // get the length of the list
     a.length should equal(res0)
```

Comment:
I think we shouldn't include `length` into the same exercise as `reverse`, or? What do you think?